test/fixtures/output/environment/11-prompt.golden

```diff
@@ -1,2 +1,3 @@
 Are you sure you want to delete environment "not-595"?
 To confirm, type "default". To cancel, press Ctrl-C: Deleted environment "not-595".
+Switch to an active environment with "confluent environment use"
```

Comment:
Any way we can put `Suggestions:` on top of this, in the same format we use for our errors? (We should convey to the user that this isn't mandatory)

Reply:
yes sure